lms/djangoapps/discussion_api/serializers.py

```diff
@@ -187,6 +187,7 @@ class ThreadSerializer(_ContentSerializer):
     non_endorsed_comment_list_url = serializers.SerializerMethodField("get_non_endorsed_comment_list_url")
     read = serializers.BooleanField(read_only=True)
     has_endorsed = serializers.BooleanField(read_only=True, source="endorsed")
+    response_count = serializers.IntegerField(source="resp_total", read_only=True)
```

Comment:
@wajeeha-khalid Note: there is a big PR that is landing any day now that upgrades our DRF version to v3.1: https://github.com/edx/edx-platform/pull/9685.  So this will be a merge conflict with that PR.

@wedaly Should we go ahead and merge this PR to master before the DRF PR merges?  If so, we'll need to update this code as part of the merge.